testproject/Assets/Scripts/GrabbableBall.cs

```diff
@@ -114,7 +114,7 @@ private void UpdateParentClientRpc(ulong ownerClientId, bool isFree)
         {
             m_CachedParent = transform.parent;
             transform.parent = playerObject.transform;
-            transform.localPosition = Vector3.up;
+            transform.localPosition = Vector3.up * (1 / playerObject.transform.localScale.y);
```

Comment:
this was needed because after we parent and move it on top of our head, it was colliding with out cube body and pushing us down which was causing PlayerCube to constantly push itself down and fall out of the world :)